doc/Comments.xml

```diff
@@ -297,6 +319,25 @@ Order(S5);
 ]]></Listing>
 </Subsection>
 
+
+<Subsection Label="@Listing">
```

Comment:
Perhaps just use @Log with a `include_in_tests` flag... but that also is confusing to people who use GAPDoc.
Of course people who do not use GAPDoc would be less affected...

Still, the very clear choice for this would be @Example, it's unfortunate this has been taken already. So... perhaps `@Demo`?  (I'd have used that for what is now called @Example, but I understand it's a bit late now to make changes here)